active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,356 @@
+        PreviousButtonVisibility="AlwaysVisible"
+        PreviousButtonText="Prev"
+        NextButtonVisibility="AlwaysVisible"
+        NextButtonText="Next"
+        LastButtonVisibility="None"
+	FirstButtonCommand="FirstButtonPressedEvent"
+        PreviousButtonCommand="PreviousButtonPressedEvent"
+        NextButtonCommand="NextButtonPressedEvent"
+        LastButtonCommand="LastButtonPressedEvent"
+        PagerInputCommand="PagerInputEvent"
+	Prefix="Page"
+	Suffix="of 10"/>
+</Grid>
+```
+
+### Numerical Button Panel Pager Control
+
+![](images/pager-control-numerical-button-panel.png)
+
+XAML
+```XAML
+<GridView x:Name="gridView1" ... />
+<controls:PagerControl x:Name="MainPagerControl"
+        DisplayMode="NumericalButtonPanel"
+	NumberOfPages="10"
+        FirstButtonVisibility="AlwaysVisible"
+        PreviousButtonVisibility="AlwaysVisible"
+        NextButtonVisibility="AlwaysVisible"
+        LastButtonVisibility="AlwaysVisible"
+        FirstButtonCommand="FirstButtonPressedEvent"
+        PreviousButtonCommand="PreviousButtonPressedEvent"
+        NextButtonCommand="NextButtonPressedEvent"
+        LastButtonCommand="LastButtonPressedEvent"
+        PagerInputCommand="PagerInputEvent"
+        EllipsisEnabled="True"
+	EllipsisShowFirstAndLast="True"
+        NumberOfIndicesShowing="6"
+	EllipsisMaxBefore="5"
+	EllipsisMaxAfter="1"/>
+</Grid>
+```
+
+### Editable NumberBox Pager Control
+
+![](images/pager-control-editable-textbox.png)
+
+XAML
+```XAML
+<GridView x:Name="gridView1" ... />
+<controls:PagerControl x:Name="MainPagerControl"
+        DisplayMode="NumberBox"
+        NumberOfPages="5"
+	FirstButtonVisibility="None"
+        PreviousButtonVisibility="AlwaysVisible"
+        NextButtonVisibility="AlwaysVisible"
+        LastButtonVisibility="None"
+	FirstButtonCommand="FirstButtonPressedEvent"
+        PreviousButtonCommand="PreviousButtonPressedEvent"
+        NextButtonCommand="NextButtonPressedEvent"
+        LastButtonCommand="LastButtonPressedEvent"
+        PagerInputCommand="PagerInputEvent"
+	Suffix="of 5"/>
+</Grid>
+```
+
+## Adding a PagerControl to Your Application
+
+Here is an example for how to add the PagerControl to your application. It can be added on the page as shown below, or can be used in the template of the layout view you prefer to use. PagerControl is added to the template for DataGrid and ItemsRepeater in WinUI 2.x and will be added to ListView and GridView in WinUI 3. 
+
+(Insert XAML and code behind example here. This will also be added to the XAML controls gallery.)
+
+
+## Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+## API Notes
```

Comment:
There should at least be one-line descriptions for any non-obvious APIs here. The one-liners become the IntelliSense text.

Reply:
I will try and commit the entire one-line description section together so you can see all of them at one time :)